packages/selected-network-controller/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [19.1.0]
+
+### Removed
+
+- The `SelectedNetworkController` constructor no longer expects a `useRequestPreference` boolean nor an `onPreferencesStateChange` listener. Removal of these parameters means that `domains` state will always be added for sites that are granted permissions. ([#4941](https://github.com/MetaMask/core/pull/4941))
```

Comment:
Similar as above, removing these options will cause type errors if they're still attempting to be passed. Should we release this under 20.0.0?